packages/core/admin/admin/src/content-manager/components/Inputs/index.js

```diff
@@ -43,8 +43,8 @@ function Inputs({
   const disabled = useMemo(() => !get(metadatas, 'editable', true), [metadatas]);
   const type = fieldSchema.type;
 
-  const errorId = useMemo(() => {
-    return get(formErrors, [keys, 'id'], null);
+  const error = useMemo(() => {
```

Comment:
We can remove the useMemo here it does not improve performances IMO.

Reply:
Done in 861ff69617e4a99415306a8fdd6779439d1f7e93